tests/jballerina-unit-test/src/test/resources/test-src/types/intersection/test_intersection_type_negative.bal

```diff
@@ -43,3 +43,9 @@ type Z record {
     X x;
     Y y;
 };
+
+function testUnsupportedIntersectionWithReadonly(stream<int> strm) {
+    (stream<int>|string:RegExp) & readonly x = strm;
```

Comment:
Use _ to avoid unused variables.

Reply:
Updated in [commit](https://github.com/ballerina-platform/ballerina-lang/pull/41843/commits/6f8cd3857971872ba1c12e95f60ebb0d4709452b).